### ISSUE TITLE
[Introspection] Fields and models starting with numbers should be valid

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1076]
Currently the parser does not like fields that start with a number. This should not be the case, as pure number column names are perfectly valid.

Problem:
https://github.com/prisma/introspection-engine-output/pull/5/files#diff-2bec550b0f9e53fd6ae9bf24b1f12bed

```
error: Unexpected token. Expected one of: End of block ("}"), field declaration.
  -->  schema.prisma:2
   | 
 1 | model Data {
 2 |   1960           Float?
   |   ^ Unexpected token.
 3 |   1961           Float?
   | 
```

But:

```
CREATE TABLE `Data` (
	`Country Name` VARCHAR(255) NULL DEFAULT NULL,
	`Country Code` VARCHAR(255) NOT NULL,
	`Indicator Name` VARCHAR(255) NULL DEFAULT NULL,
	`Indicator Code` VARCHAR(255) NOT NULL,
	`1960` DOUBLE NULL DEFAULT NULL,
	`1961` DOUBLE NULL DEFAULT NULL,
	`1962` DOUBLE NULL DEFAULT NULL,
	`1963` DOUBLE NULL DEFAULT NULL,
	`1964` DOUBLE NULL DEFAULT NULL,
	`1965` DOUBLE NULL DEFAULT NULL,
	`1966` DOUBLE NULL DEFAULT NULL,
	`1967` DOUBLE NULL DEFAULT NULL,
	`1968` DOUBLE NULL DEFAULT NULL,
	`1969` DOUBLE NULL DEFAULT NULL,
	`1970` DOUBLE NULL DEFAULT NULL,
	`1971` DOUBLE NULL DEFAULT NULL,
	`1972` DOUBLE NULL DEFAULT NULL,
	`1973` DOUBLE NULL DEFAULT NULL,
	`1974` DOUBLE NULL DEFAULT NULL,
	`1975` DOUBLE NULL DEFAULT NULL,
	`1976` DOUBLE NULL DEFAULT NULL,
	`1977` DOUBLE NULL DEFAULT NULL,
	`1978` DOUBLE NULL DEFAULT NULL,
	`1979` DOUBLE NULL DEFAULT NULL,
	`1980` DOUBLE NULL DEFAULT NULL,
	`1981` DOUBLE NULL DEFAULT NULL,
	`1982` DOUBLE NULL DEFAULT NULL,
	`1983` DOUBLE NULL DEFAULT NULL,
	`1984` DOUBLE NULL DEFAULT NULL,
	`1985` DOUBLE NULL DEFAULT NULL,
	`1986` DOUBLE NULL DEFAULT NULL,
	`1987` DOUBLE NULL DEFAULT NULL,
	`1988` DOUBLE NULL DEFAULT NULL,
	`1989` DOUBLE NULL DEFAULT NULL,
	`1990` DOUBLE NULL DEFAULT NULL,
	`1991` DOUBLE NULL DEFAULT NULL,
	`1992` DOUBLE NULL DEFAULT NULL,
	`1993` DOUBLE NULL DEFAULT NULL,
	`1994` DOUBLE NULL DEFAULT NULL,
	`1995` DOUBLE NULL DEFAULT NULL,
	`1996` DOUBLE NULL DEFAULT NULL,
	`1997` DOUBLE NULL DEFAULT NULL,
	`1998` DOUBLE NULL DEFAULT NULL,
	`1999` DOUBLE NULL DEFAULT NULL,
	`2000` DOUBLE NULL DEFAULT NULL,
	`2001` DOUBLE NULL DEFAULT NULL,
	`2002` DOUBLE NULL DEFAULT NULL,
	`2003` DOUBLE NULL DEFAULT NULL,
	`2004` DOUBLE NULL DEFAULT NULL,
	`2005` DOUBLE NULL DEFAULT NULL,
	`2006` DOUBLE NULL DEFAULT NULL,
	`2007` DOUBLE NULL DEFAULT NULL,
	`2008` DOUBLE NULL DEFAULT NULL,
	`2009` DOUBLE NULL DEFAULT NULL,
	`2010` DOUBLE NULL DEFAULT NULL,
	`2011` DOUBLE NULL DEFAULT NULL,
	PRIMARY KEY (`Country Code`, `Indicator Code`),
	INDEX `Indicator Code` (`Indicator Code`),
	INDEX `Country Code` (`Country Code`),
	CONSTRAINT `Data_ibfk_1` FOREIGN KEY (`Indicator Code`) REFERENCES `Metadata - Indicators` (`INDICATOR_CODE`) ON UPDATE CASCADE ON DELETE CASCADE,
	CONSTRAINT `Data_ibfk_2` FOREIGN KEY (`Country Code`) REFERENCES `Metadata - Countries` (`Country Code`) ON UPDATE CASCADE ON DELETE CASCADE
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB
ROW_FORMAT=COMPACT
;

```

![image](https://user-images.githubusercontent.com/183673/70395815-f58aea00-1a02-11ea-9c1f-9e2bf3d98982.png)

**Comment by janpio** [comment]
More:

```
error: Unexpected token. Expected one of: End of block ("}"), field declaration.
  -->  schema.prisma:2
   | 
 1 | model Actions {
 2 |   3PointAttempts    Int?
   |   ^ Unexpected token.
 3 |   3PointsMade       Int?
   | 
```
https://github.com/prisma/introspection-engine-output/pull/5/files#diff-5d695f835ad68fa7b749fc0905456b38

**Comment by janpio** [comment]
The same should be valid for tables/models:

Error:
```
error: Unexpected token. Expected one of: alphanumeric identifier.
  -->  schema.prisma:1
   | 
   | 
 1 | model 20news {
   |       ^ Unexpected token.
 2 |   id    Int     @id
   | 
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-3ced6a5183fe2f8c2f1d4642ad883643)

Model:
```
model 20news {
  id    Int     @id
  label String?
  text  String?
}
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-89648fad88f01ab59f1f59e826c51939R3-R7)

Original DB:
```
CREATE TABLE `20news` (
	`id` INT(11) NOT NULL,
	`text` MEDIUMTEXT NULL,
	`label` VARCHAR(24) NULL DEFAULT NULL,
	PRIMARY KEY (`id`)
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB
;

```

![image](https://user-images.githubusercontent.com/183673/70396845-e14bea80-1a0c-11ea-9fae-426679589121.png)

**Comment by pantharshit00** [comment]
I can confirm this. The workaround here would be to add a `@map` directive manually.

**Comment by janpio** [comment]
fixed via #1139
